finance/portfolio-performance/Portfile

```
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               java 1.0
+
+github.setup            buchen portfolio 0.50.3
+name                    portfolio-performance
+categories              finance
+platforms               darwin
+license                 EPL-1
+maintainers             {@janosch mailbox.org:janosch1} openmaintainer
+description             A simple tool to calculate the overall performance \
+                        of an investment portfolio.
+long_description        An open source tool to calculate the overall performance \
+                        of an investment portfolio - across all accounts - using \
+                        True-Time Weighted Return or Internal Rate of Return.
+homepage                https://www.portfolio-performance.info/en/
+
+checksums               rmd160  2b6485f607493ffff233dedbc7f68191d442a10c \
+                        sha256  f4d8d348e28b4e8f337ddb282f0f7cbce92fbe4190bdfec5fcc4ca92a83833a4 \
+                        size    3238967
+
+supported_archs         x86_64
+java.version            11
```

```suggestion
java.version            11+
java.fallback           openjdk11
```

Can we allow later versions of Java? Reading throught the trac ticket, I see there are some issues with asterisk and plus notation, so please correct me if this wouldn't work with the current bug.

I would also suggest adding an LTS JDK port as a fallback.

Otherwise, lgtm. Thanks @Janosch.

---

@harens It does not compile out of the box with higher Java versions. With Java 15, for example, I had to increase the version of _tycho_, a dependency of _portfolio-performance_. As the github page of _portfolio-performance_ states Java 11 explicitly as a prerequisite, I'd like to stick with 11 (without "+").

I added the java.fallback as suggested by you.

---

And yes, 11+ does not work on Big Sur at the moment. It will install the fallback even if you have JDK 15 installed.